### ISSUE TITLE
Tidying up dates on What's On

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#2660]
## Type
- 🚑 Health report

Now that we've defaulted to Everything on What's On, the start date seems a lot less relevant and useful. Is there any user need for this? If not, can we simplify by getting rid of it? @Heesoomoon 

We might wish to retain `Friday-Sunday` on the This Weekend view

![image](https://user-images.githubusercontent.com/1446139/40308503-6ab18726-5cfe-11e8-9f16-604d96e3b3db.png)

**Comment by Heesoomoon** [comment]
Yes, we can remove it but could we wait till we revisit the header for all states of What's on to resolve the display logic of the relevant info? e.g. date, date range, etc.
